gudrun_classes/container.py

```diff
@@ -57,7 +61,8 @@ def __init__(self):
         self.geometry = ""
         self.thickness = (0.0, 0.0)
         self.angleOfRotationSampleWidth = (0.0, 0.0)
-        self.densityOfAtoms = 0.0
+        self.density = 0.0
+        self.densityUnits = 0
```

Comment:
Since we're now using an enum here, I would prefer to see this as:
```suggestion
        self.densityUnits = UnitsOfDensity.ATOMIC
```